aten/CMakeLists.txt

```diff
@@ -1,12 +1,19 @@
-cmake_minimum_required(VERSION 3.0)
-set(CMAKE_MODULE_PATH
-  ${CMAKE_CURRENT_SOURCE_DIR}/cmake
-  ${CMAKE_CURRENT_SOURCE_DIR}/../cmake/Modules_CUDA_fix
+if (CAFFE2_CMAKE_BUILDING_WITH_MAIN_REPO)
```

Comment:
What is the planned terminal name for this variable?

Reply:
Yep, this is used in modules. In the future we might want to call it something like "BUILDING_AS_STANDALONE_PROJECT" but it really doesn't matter.